vscode extension packaging broken

```
$ make package
Creating .vsix package...
npm run package

> kb-dashboard-compiler@0.1.10 package
> vsce package

Executing prepublish script 'npm run vscode:prepublish'...

> kb-dashboard-compiler@0.1.10 vscode:prepublish
> npm run package-bundle


> kb-dashboard-compiler@0.1.10 package-bundle
> node esbuild.js --production

[watch] build started
✘ [ERROR] Could not resolve "yaml"
    src/esqlQueryExtractor.ts:8:88:
[watch] build finished
Error: Build failed with 1 error:
src/esqlQueryExtractor.ts:8:88: ERROR: Could not resolve "yaml"
    at failureErrorWithLog (/Users/gil/src/elastic/kb-yaml-to-lens/vscode-extension/node_modules/esbuild/lib/main.js:1467:15)
    at /Users/gil/src/elastic/kb-yaml-to-lens/vscode-extension/node_modules/esbuild/lib/main.js:926:25
    at /Users/gil/src/elastic/kb-yaml-to-lens/vscode-extension/node_modules/esbuild/lib/main.js:1345:9
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5) {
  errors: [Getter/Setter],
  warnings: [Getter/Setter]
}
 ERROR  npm failed with exit code 1
make: *** [package] Error 1
```

--- a/vscode-extension/Makefile
+++ b/vscode-extension/Makefile
@@ -30,58 +30,34 @@ install:
 
 all: ci
 
-ci:
-	@if [ ! -d "node_modules" ]; then \
-		echo "⚠ Dependencies not installed, installing..."; \
-		$(MAKE) install; \
-	fi; \
-	$(MAKE) compile lint test-unit; \
-	echo "✓ VSCode extension CI checks passed"
+ci: install
+	$(MAKE) compile lint test-unit
+	@echo "✓ VSCode extension CI checks passed"
 
 check: ci
 
-fix:
-	@if [ ! -d "node_modules" ]; then \
-		echo "⚠ Dependencies not installed, installing..."; \
-		$(MAKE) install; \
-	fi; \
-	echo "Auto-fixing ESLint issues..."; \
+fix: install
+	@echo "Auto-fixing ESLint issues..."
 	npm run lint -- --fix
 
-test:
-	@if [ ! -d "node_modules" ]; then \
-		echo "⚠ Dependencies not installed, installing..."; \
-		$(MAKE) install; \
-	fi; \
-	echo "Running VSCode extension tests..."; \
+test: install
+	@echo "Running VSCode extension tests..."
 	npm test
 
-test-unit:
-	@if [ ! -d "node_modules" ]; then \
-		echo "⚠ Dependencies not installed, installing..."; \
-		$(MAKE) install; \
-	fi; \
-	echo "Running TypeScript tests for VSCode extension..."; \
+test-unit: install
+	@echo "Running TypeScript tests for VSCode extension..."
 	npm run compile && npm run test:unit
 
-compile:
-	@if [ ! -d "node_modules" ]; then \
-		echo "⚠ Dependencies not installed, installing..."; \
-		$(MAKE) install; \
-	fi; \
-	echo "Compiling TypeScript..."; \
+compile: install
+	@echo "Compiling TypeScript..."
 	npm run compile
 
-watch:
+watch: install
 	@echo "Starting watch mode..."
 	npm run watch
 
-lint:
-	@if [ ! -d "node_modules" ]; then \
-		echo "⚠ Dependencies not installed, installing..."; \
-		$(MAKE) install; \
-	fi; \
-	echo "Running ESLint..."; \
+lint: install
+	@echo "Running ESLint..."
 	npm run compile && npm run lint
 
 lint-check: lint

